arch/arm64/boot/dts/qcom/sdm660-xiaomi-lavender.dts

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2020, The Linux Foundation. All rights reserved.
```

Comment:
I would usually copy this from something I used as a reference and keep the date as-is. Unless you are involved with the Linux foundation. Otherwise it implies (to me at least) that the Linux Foundation has worked on it in 2020.

Reply:
Should I just delete mention of Linux Foundation? This file was written by me from scratch :)

Reply:
I guess so, yes.

Reply:
:ok: